design/process/review-guidelines.md

```diff
@@ -8,40 +8,55 @@ editor: markdown
 dateCreated: 2021-05-28T21:02:49.613Z
 ---
 
-The design team posts new specs to [GitHub Discussions](https://github.com/centerofci/mathesar/discussions/categories/review-request) when there are new designs to review. All [core team](/team) members are encouraged to review new designs. This page describes the process.
+The design team creates specs to describe the design solutions related to the Mathesar product. Whether small or large, all design issues should be accompanied by a spec containing the solution details and information relevant to its implementation. Every spec goes through a review process so that the team can discuss it and ask questions.
 
-# Review Process
+The spec creation process begins when a design issue is started and finalizes when the spec is approved and merged. When a design issue is ready for review, a spec is created, and a PR is opened. Finalizing a design spec means, in most cases, that the design issue has also been solved. If this is not the case, the reason and additional requirements must be communicated in the design issue and the spec.
 
-## Authors
-- Write your design spec on the wiki under the [Documents](/design/specs) folder and link to it from the top-level [Documents](/design/specs) page.
+## Review Process
+
+### Authors
+
+- Clone the [Mathesar Wiki Repo](https://github.com/centerofci/mathesar-wiki) and create a new page under the [Design Specs](/design/specs) directory. Additionally, create a link to it from the top-level [Documents](/design/specs) page.
   - Please follow the format of the most recent spec. Spec formats may change over time.
-- Once you think the spec is ready for review, create a GitHub Discussion that includes:
-  - the design document and all related resources such as wireframes, prototypes, etc. 
+- Add the [spec content](##spec-content) to the page. Make sure the page date is updated as well.
+- Once you think the spec is ready for review, create a PR that includes:
+  - a link to the referenced design issue from the [Mathesar Repo](https://github.com/centerofci/mathesar)
```

Comment:
This should be listed as "Fixes `<link to the issue>`" so that the issue will be automatically closed when the PR is merged.